extensions/measurement-tracking/src/panels/PanelMeasurementTableTracking/index.js

```diff
@@ -53,25 +59,33 @@ function PanelMeasurementTableTracking({ servicesManager, extensionManager }) {
     debouncedMeasurementChangeTimestamp,
   ]);
 
-  // ~~ DisplayStudySummary
-  useEffect(() => {
+  const updateDisplayStudySummary = async () => {
     if (trackedMeasurements.matches('tracking')) {
       const StudyInstanceUID = trackedStudy;
       const studyMeta = DicomMetadataStore.getStudy(StudyInstanceUID);
       const instanceMeta = studyMeta.series[0].instances[0];
-      const { Modality, StudyDate, StudyDescription } = instanceMeta;
+      const { StudyDate, StudyDescription } = instanceMeta;
+
+      const modalities = new Set();
+      studyMeta.series.forEach(series => modalities.add(series.instances[0].Modality));
```

Comment:
This should be filtered by `trackedStudies`. I'll create a follow-up issue

Reply:
@dannyrb it would be nice if you could explain this a little more in the ticket, i don't have much background / context.